src/System.Management.Automation/engine/remoting/common/remotingexceptions.cs

```diff
@@ -67,7 +67,7 @@ internal enum PSRemotingErrorId : uint
         MissingIsStartFragment = 409,
         MissingProperty = 410,
         ObjectIdsNotMatching = 411,
-        FragmetIdsNotInSequence = 412,
```

Comment:
`PSRemotingErrorId` is tightly coupled with `RemotingErrorIdStrings`. We need to change them together.